<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="5" passed="2">
  <reporter-output>
    <line>
      <![CDATA[The Title of the Page is matching]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[The Title of the Page is matching]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[The Element is not present]]>
    </line>
    <line>
      <![CDATA[The Title of the Page is matching]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[Displayed Valid Email and Invalid pop up]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[The Title of the Page is matching]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[The Element is present]]>
    </line>
    <line>
      <![CDATA[Inavild mail or password pop up not generated]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="247572" started-at="2018-02-10T16:37:05Z" finished-at="2018-02-10T16:41:13Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="247572" started-at="2018-02-10T16:37:05Z" finished-at="2018-02-10T16:41:13Z">
      <class name="scripts.ValidLoginLogout">
        <test-method status="PASS" signature="openApplication()[pri:0, instance:scripts.ValidLoginLogout@1ce92674]" name="openApplication" is-config="true" duration-ms="27115" started-at="2018-02-10T16:37:26Z" finished-at="2018-02-10T16:37:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApplication -->
        <test-method status="FAIL" signature="LoginTest()[pri:0, instance:scripts.ValidLoginLogout@1ce92674]" name="LoginTest" duration-ms="47705" started-at="2018-02-10T16:37:53Z" finished-at="2018-02-10T16:38:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at generic.Base_Page.elementIsPresent(Base_Page.java:48)
at pom.LoginPage.verifyLoginpage(LoginPage.java:235)
at scripts.ValidLoginLogout.LoginTest(ValidLoginLogout.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[The Title of the Page is matching]]>
            </line>
            <line>
              <![CDATA[The Element is present]]>
            </line>
            <line>
              <![CDATA[The Element is present]]>
            </line>
            <line>
              <![CDATA[The Element is not present]]>
            </line>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="closeApp()[pri:0, instance:scripts.ValidLoginLogout@1ce92674]" name="closeApp" is-config="true" duration-ms="766" started-at="2018-02-10T16:38:41Z" finished-at="2018-02-10T16:38:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- scripts.ValidLoginLogout -->
      <class name="scripts.ForgotPasswordTest">
        <test-method status="PASS" signature="openApplication()[pri:0, instance:scripts.ForgotPasswordTest@5700d6b1]" name="openApplication" is-config="true" duration-ms="56649" started-at="2018-02-10T16:39:10Z" finished-at="2018-02-10T16:40:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApplication -->
        <test-method status="PASS" signature="ForgotPassword()[pri:0, instance:scripts.ForgotPasswordTest@5700d6b1]" name="ForgotPassword" duration-ms="3450" started-at="2018-02-10T16:40:07Z" finished-at="2018-02-10T16:40:10Z">
          <reporter-output>
            <line>
              <![CDATA[The Element is present]]>
            </line>
            <line>
              <![CDATA[The Element is present]]>
            </line>
            <line>
              <![CDATA[The Element is present]]>
            </line>
          </reporter-output>
        </test-method> <!-- ForgotPassword -->
        <test-method status="PASS" signature="closeApp()[pri:0, instance:scripts.ForgotPasswordTest@5700d6b1]" name="closeApp" is-config="true" duration-ms="720" started-at="2018-02-10T16:40:10Z" finished-at="2018-02-10T16:40:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- scripts.ForgotPasswordTest -->
      <class name="scripts.InvalidEmailPopUpmessage">
        <test-method status="PASS" signature="openApplication()[pri:0, instance:scripts.InvalidEmailPopUpmessage@3fee9989]" name="openApplication" is-config="true" duration-ms="19590" started-at="2018-02-10T16:37:05Z" finished-at="2018-02-10T16:37:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApplication -->
        <test-method status="FAIL" signature="InvalidEmailPopUptest()[pri:0, instance:scripts.InvalidEmailPopUpmessage@3fee9989]" name="InvalidEmailPopUptest" duration-ms="328" started-at="2018-02-10T16:37:25Z" finished-at="2018-02-10T16:37:25Z">
          <exception class="org.openqa.selenium.InvalidElementStateException">
            <message>
              <![CDATA[invalid element state: Element is not currently interactable and may not be manipulated
  (Session info: chrome=64.0.3282.119)
  (Driver info: chromedriver=2.33.506120 (e3e53437346286c0bc2d2dc9aa4915ba81d9023f),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'DESKTOP-L3RI81U', ip: '192.168.0.102', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptSslCerts: true, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.33.506120 (e3e53437346286..., userDataDir: C:\Users\OPTIME~1\AppData\L...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 64.0.3282.119, webStorageEnabled: true}
Session ID: 5a9590c3ae2bf38d36bb7d504f78a507]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.InvalidElementStateException: invalid element state: Element is not currently interactable and may not be manipulated
  (Session info: chrome=64.0.3282.119)
  (Driver info: chromedriver=2.33.506120 (e3e53437346286c0bc2d2dc9aa4915ba81d9023f),platform=Windows NT 10.0.16299 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 0 milliseconds
Build info: version: '3.9.1', revision: '63f7b50', time: '2018-02-07T22:25:02.294Z'
System info: host: 'DESKTOP-L3RI81U', ip: '192.168.0.102', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptSslCerts: true, applicationCacheEnabled: false, browserConnectionEnabled: false, browserName: chrome, chrome: {chromedriverVersion: 2.33.506120 (e3e53437346286..., userDataDir: C:\Users\OPTIME~1\AppData\L...}, cssSelectorsEnabled: true, databaseEnabled: false, handlesAlerts: true, hasTouchScreen: false, javascriptEnabled: true, locationContextEnabled: true, mobileEmulationEnabled: false, nativeEvents: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, rotatable: false, setWindowRect: true, takesHeapSnapshot: true, takesScreenshot: true, unexpectedAlertBehaviour: , unhandledPromptBehavior: , version: 64.0.3282.119, webStorageEnabled: true}
Session ID: 5a9590c3ae2bf38d36bb7d504f78a507
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)
at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)
at org.openqa.selenium.remote.http.JsonHttpResponseCodec.reconstructValue(JsonHttpResponseCodec.java:40)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:80)
at org.openqa.selenium.remote.http.AbstractHttpResponseCodec.decode(AbstractHttpResponseCodec.java:44)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:160)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:279)
at org.openqa.selenium.remote.RemoteWebElement.clear(RemoteWebElement.java:118)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at com.sun.proxy.$Proxy9.clear(Unknown Source)
at pom.LoginPage.enterEmail(LoginPage.java:169)
at scripts.InvalidEmailPopUpmessage.InvalidEmailPopUptest(InvalidEmailPopUpmessage.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.InvalidElementStateException -->
          <reporter-output>
            <line>
              <![CDATA[The Title of the Page is matching]]>
            </line>
            <line>
              <![CDATA[The Element is present]]>
            </line>
          </reporter-output>
        </test-method> <!-- InvalidEmailPopUptest -->
        <test-method status="PASS" signature="closeApp()[pri:0, instance:scripts.InvalidEmailPopUpmessage@3fee9989]" name="closeApp" is-config="true" duration-ms="713" started-at="2018-02-10T16:37:25Z" finished-at="2018-02-10T16:37:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- scripts.InvalidEmailPopUpmessage -->
      <class name="scripts.AlreadyRegisteredUserPop">
        <test-method status="PASS" signature="openApplication()[pri:0, instance:scripts.AlreadyRegisteredUserPop@73ad2d6]" name="openApplication" is-config="true" duration-ms="24571" started-at="2018-02-10T16:38:41Z" finished-at="2018-02-10T16:39:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApplication -->
        <test-method status="PASS" signature="SignUp()[pri:0, instance:scripts.AlreadyRegisteredUserPop@73ad2d6]" name="SignUp" duration-ms="3289" started-at="2018-02-10T16:39:06Z" finished-at="2018-02-10T16:39:09Z">
          <reporter-output>
            <line>
              <![CDATA[The Title of the Page is matching]]>
            </line>
            <line>
              <![CDATA[The Element is present]]>
            </line>
            <line>
              <![CDATA[The Element is present]]>
            </line>
            <line>
              <![CDATA[Displayed Valid Email and Invalid pop up]]>
            </line>
          </reporter-output>
        </test-method> <!-- SignUp -->
        <test-method status="PASS" signature="closeApp()[pri:0, instance:scripts.AlreadyRegisteredUserPop@73ad2d6]" name="closeApp" is-config="true" duration-ms="773" started-at="2018-02-10T16:39:09Z" finished-at="2018-02-10T16:39:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- scripts.AlreadyRegisteredUserPop -->
      <class name="scripts.SignUPpopup">
        <test-method status="PASS" signature="openApplication()[pri:0, instance:scripts.SignUPpopup@7085bdee]" name="openApplication" is-config="true" duration-ms="57809" started-at="2018-02-10T16:40:11Z" finished-at="2018-02-10T16:41:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApplication -->
        <test-method status="FAIL" signature="SignUp()[pri:0, instance:scripts.SignUPpopup@7085bdee]" name="SignUp" duration-ms="3256" started-at="2018-02-10T16:41:09Z" finished-at="2018-02-10T16:41:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at generic.Base_Page.popmesg(Base_Page.java:69)
at scripts.SignUPpopup.SignUp(SignUPpopup.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[The Title of the Page is matching]]>
            </line>
            <line>
              <![CDATA[The Element is present]]>
            </line>
            <line>
              <![CDATA[The Element is present]]>
            </line>
            <line>
              <![CDATA[Inavild mail or password pop up not generated]]>
            </line>
          </reporter-output>
        </test-method> <!-- SignUp -->
        <test-method status="PASS" signature="closeApp()[pri:0, instance:scripts.SignUPpopup@7085bdee]" name="closeApp" is-config="true" duration-ms="755" started-at="2018-02-10T16:41:12Z" finished-at="2018-02-10T16:41:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- scripts.SignUPpopup -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
